packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -469,6 +469,27 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		);
 	}
 
+	/**
+	 * Prints a message to the console.
+	 * @param message The message to print.
+	 */
+	async helloWorld(message: string) {
+		const phpResponse = await this.run({
+			code: `
```

Comment:
```suggestion
			code: `<?php
```

Otherwise it will output a some whitespace before the message :-)